doc/tech/approach.md

```diff
@@ -80,27 +80,19 @@ To address the concern areas detailed above the specific topics listed below wil
-
-•	**Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
--	Control, User and Management traffic types
--	Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
-
-•	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
-
-•	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
-
-•	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
-
-•	**Scalability**: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
-
-•	**Workload diversity**: Should be capable to p=to provide connectivity to any type of workloads, including VNF , CNF and BareMetal workloads.
-
-•	**Telco Grade**: Should support deployments with Telco Grade characteristics requirements.
-
-•	**Future proof**: Should support known and emerging technology trends e.g. SmartNICs, FPGAs and Programmable Switches, multi-cloud aspects (public/private/hybrid), multi-location aspects (multi-cluster, DCI, etc), Edge realted technologies, etc.
-
+- **Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+  - Control, User and Management traffic types
+  - Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
+-	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
+-	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
+-	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
+-	**Scalability**: Should be scalable to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+- **Workload diversity**: Should be capable to provide connectivity to any type of workloads, including VNF , CNF and BareMetal workloads.
+-	**Telco Grade**: Should support deployments with Telco Grade characteristics requirements.
+-	**Future proof**: Should support known and emerging technology trends e.g. SmartNICs, FPGAs and Programmable Switches, multi-cloud aspects (public/private/hybrid), multi-location aspects (multi-cluster, DCI, etc), Edge realted technologies, etc.
+- **Dimensionability**: Should support network design in a way that the network is capable to handle the traffic in a non-blocking manner without swapping out the leaf switches.
```

Comment:
"leaf switches" implies a particular networking architecture. In the RM should we keep away from such decisions.
sentence simplification
```suggestion
- **Dimensionability**: Should support networks capable of handling the traffic in a non-blocking manner without swapping out networking hardware.
```

Reply:
Didn't we plan to merge scalability and dimensionability? 
What about:
**Scalability** Should be scalable to handle all traffic traverse North-South and East-West enabling small to large deployments in a non-blocking manner.